spec/vcr_helper.rb

```diff
@@ -46,3 +47,23 @@
   record: ENV['VCR_OPTS_RECORD'].try!(:to_sym) || :none,
   allow_unused_http_interactions: false
 }
+
+# VCR can update content length headers to solve various problems caused
+# by recording and playing back requests and responses.  The update process
+# happens in a before_playback hook.  Filtering senstive data also occurs
+# in before hooks.  Since there isn't a way to make sure that the content
+# length update happens after filtering senstive data (and since filtering
+# data can mess with the content length), we end up not getting the benefit
+# out of the update_content_length_header hook that we are shooting for.
+# This monkey patch fixes this by forcing a content length update after
+# every filtering of sensitive data.  This does more work than we need
+# but it is also fast so it isn't that big of a deal.
+
+class VCR::HTTPInteraction::HookAware
+  alias_method :orig_filter!, :filter!
```

Comment:
here..you're not using blank line after class and before class end.  so this seems a bit more normal to me.  but just let me know ;)